docs/introduction/integration_test.rst

```diff
@@ -349,7 +349,8 @@ gives access to the following parameters:
 - case_id : current test case identification number (optional for test suite setup and teardown)
 - aux_list : list of used auxiliaries
 
-Based on Message Protocol, users can configure the maximum time (in seconds) used to wait for a report.
+If GreyTestCase / GreyTestSuite are used, based on Message Protocol, 
```

Comment:
This is too small to understand the implication of using the GreyTest approach. We probably need to cluster the documentation differently. Some of the current documentation has the GreyTest (former basicTest) as "defacto" which is, after merging this PR not the case anymore.

What is really important is that all the tests internally still run as expected. Were the tests done?

Reply:
Yes, I am currently reworking the documentation so it describe only the BasicTest behaviour and I am structuring the GreyTest explanations into a separate section.

Reply:
Add Introduction for   Grey Tests / Testapp 

and maybe add a section when to use what.
If you use the testapp concept use greytest
if you just want to write tests with auxiliaries and connectors use basicTest

IMHO
We should have named the GreyTest to Testapp. 
For me its more intuitive if i read 
TestAppTestCase / TestAppTestSuite instead of  GreyTestCase / GreyTestSuite

GreyTest rather names the concept which the TestApp implements.

Reply:
What I went with is to add a new section TestApp that explain what it does / how to do it (use greytest / dut aux etc...) and then remove all mentions of it the introduction that describe the new default behaviour. 

I agree that Testapp is less confusing than GreyTest